fix: Reduce margin in RecipePageInstructions

felixschndr opened the pull request:
## What this PR does / why we need it:

Fixes spacing of the ingredient linker. Since Nuxt3 there was a lot of space between ingredients, which becomes a problem for recipes with many ingredients and/or lower resolution screens.

Before:
<img width="609" height="847" alt="image" src="https://github.com/user-attachments/assets/3d40c17e-3879-46ca-82cc-eff5535f7de4" />
<img width="600" height="887" alt="image" src="https://github.com/user-attachments/assets/6a6fdbdd-efe5-4d4d-9cc0-e0f05cb27529" />
<img width="608" height="912" alt="image" src="https://github.com/user-attachments/assets/a5fb463e-4789-4742-aa88-48975138ae14" />


After:
<img width="609" height="590" alt="image" src="https://github.com/user-attachments/assets/9b73c520-8e02-4e7c-a819-85356438fb39" />
<img width="609" height="638" alt="image" src="https://github.com/user-attachments/assets/e7a5384c-76e5-4672-ba6c-6dc46e037d6c" />
<img width="614" height="646" alt="image" src="https://github.com/user-attachments/assets/0c73b5de-03cb-42b5-9f55-4fea1c780832" />


## Which issue(s) this PR fixes:

-


## Testing

`task ui:lint` runs sucessfully

felixschndr committed:
Reduce margins in RecipePageInstructions between checkboxes

felixschndr committed:
Reduce margins in RecipePageInstructions between checkboxes

Kuchenpirat reviewed (commented):
You can set the density prop on v-checkbox to compact, so you don't have to use negative margins

https://vuetifyjs.com/en/components/checkboxes/#api

felixschndr commented:
@Kuchenpirat that does not seem to work as I would expect it. This is with `density="compact"` instead of `class="mb-n10 mt-n2"`
<img width="607" height="803" alt="image" src="https://github.com/user-attachments/assets/8c76be4f-fb5e-4768-bc14-260c930f981d" />

felixschndr commented:
Found the solution [here](https://github.com/vuetifyjs/vuetify/issues/15577#issuecomment-1208312444) and implemented it. I updated the images above. Ready to be reviewed @Kuchenpirat

Kuchenpirat committed:
Merge branch 'mealie-next' into fix/spacing-of-ingredient-linker

Kuchenpirat reviewed (changes requested):
Lets remove the negative margings from the checkbox components. The elements are a bit too close together for touch screen usage.

felixschndr commented:
Removed the margins and updated the screenshots above.

Kuchenpirat reviewed (approved):
LGTM 👍
Thanks for the contribution
